rules/windows/process_creation/proc_creation_win_susp_openas_rundll_usage.yml

```diff
@@ -0,0 +1,27 @@
+title: Suspicious Usage Of OpenAs_RunDLL
+id: d87bd452-6da1-456e-8155-7dc988157b7d
+description: Detects suspicious usage of the OpenAs_RunDLL function to launch other commands as seen in the the raspberry-robin attack
+status: experimental
+references:
+    - https://redcanary.com/blog/raspberry-robin/
+author: Nasreddine Bencherchali
+date: 2022/09/01
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_openasrundll:
+        CommandLine|contains: ',OpenAs_RunDLL'
```

Comment:
In the ref find only `,ShellExec_RunDLLA` , I have miss the section ?

Reply:
Yes you're correct, my mistake